api/types/kustomization.go

```diff
@@ -172,6 +172,31 @@ type Kustomization struct {
 	BuildMetadata []string `json:"buildMetadata,omitempty" yaml:"buildMetadata,omitempty"`
 }
 
+const (
+	deprecatedBaseWarningMessage           = "# Warning: bases was deprecated, Please use resources or try `kustomize edit fix`."
```

Comment:
nit: quote the field names (here, and in the other messages as well)

```
# Warning: bases was deprecated, please use 'resources' or try `kustomize edit fix`.
```

Reply:
I fixed it.